src/common/android/androidPlatform.ts

```diff
@@ -137,7 +142,7 @@ export class AndroidPlatform extends GeneralMobilePlatform {
                     if (apiVersion >= AndroidAPILevel.LOLLIPOP) { // If we support adb reverse
                         return this.adb.reverseAdd(device.id, Packager.DEFAULT_PORT.toString(), this.runOptions.packagerPort);
                     } else {
-                        Log.logWarning(`Device ${device.id} supports only API Level ${apiVersion}. `
+                        return Log.logWarning(`Device ${device.id} supports only API Level ${apiVersion}. `
```

Comment:
Though this is correct expression, it's semantically incorrect to "return log warning". Let's change this to a single `return` on a separate line;